docs/post_installation/peering.md

```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
-format (also known as `multiaddr`) to achieve that.
+In order to communicate to your peer neighbors, you will need an address to reach them.  To achieve that, Hornet uses the `MultiAddresses` format (also known as `multiaddr`).
 
-`multiAddr` is a convention how to encode multiple layers of addressing information into a single path structure that is
-future-proof. In other words, `multiaddr` is able to combine several different pieces of information in a single
-human-readable and machine-optimized string, including network protocol and [`PeerId`](#node-identity).
+`multiAddr` is a convention on how to encode multiple layers of addressing information into a single path structure that is future-proof. In other words, `multiaddr` is able to combine several pieces of information in a single human-readable and machine-optimized string, including network protocol and [`PeerId`](#node-identity).
 
 For example, a node is reachable using IPv4 `100.1.1.1` using `TCP` on port `15600` and its `PeerId`
-is `12D3KooWHjcCgWPnUEP8wNdbL2fx63Cmosk16xyZ25iUZagxmHb4`.
-
-A `multiaddr` encoding such information would look like this:
+is `12D3KooWHjcCgWPnUEP8wNdbL2fx63Cmosk16xyZ25iUZagxmHb4`. A `multiaddr` encoding of this information would look like this:
 
 ```plaintext
 /ip4/100.1.1.1/tcp/15600/p2p/12D3KooWHjcCgWPnUEP8wNdbL2fx63Cmosk16xyZ25iUZagxmHb4
 ```
 
-> Note how `ip4` is used. A common mistake is to use `ipv4`.
+:::info
+Note how `ip4` is used. A common mistake is to use `ipv4`.
+:::
 
 If a node is reachable using a DNS name (for example `node01.iota.org`), then the given `multiaddr` would be:
 
 ```plaintext
 /dns/node01.iota.org/tcp/15600/p2p/12D3KooWHjcCgWPnUEP8wNdbL2fx63Cmosk16xyZ25iUZagxmHb4
 ```
 
-In order to find out your own `multiaddr` to give to your peers for neighboring, combine the `peerId` you have gotten
-from the stdout when the Hornet node started up (or which was shown via the `p2pidentity` CLI tool) and your
-configured `p2p.bindAddress`. Obviously replace the `/ip4/<ip_address>`/`/dns/<hostname>` segments with the actual
-information.
+You will need to find out your own `multiaddr` to give to your peers for neighboring. To do so, combine the `peerId` you have gotten
+from the stdout when the Hornet node started up (or which was shown via the `p2pidentity` CLI tool), and your
+configured `p2p.bindAddress`. Replace the `/ip4/<ip_address>`/`/dns/<hostname>` segments with the actual information.
+
+You can find more information about `multiaddr` is available at the [libp2p docs page](https://docs.libp2p.io/concepts/addressing/).
 
-More information about `multiaddr` is available at the [libp2p docs page](https://docs.libp2p.io/concepts/addressing/).
+## Adding Node Peers
 
-### Adding node peers
+Once you know your node's own `multiaddr`, it can be exchanged with other node owners to establish a mutual peer connection. We recommended number of peer neighbors between 4-6 to get some degree of redundancy.
```

Comment:
```suggestion
Once you know your node's own `multiaddr`, it can be exchanged with other node owners to establish a mutual peer connection. We recommended a number of peer neighbors between 4-6 to get some degree of redundancy.
```